server/request.js

```diff
@@ -1,11 +1,11 @@
 import axios from 'axios';
 
-export default function makeRequest(url) {
-    return axios.get(url)
-        .then(function(response){
+export default function makeRequest(url, headers) {
+    return axios.get(url, {headers: headers})

```

Comment:
There's a nice shorthand for this sort of thing, which I would recommend:
```suggestion
    return axios.get(url, {headers})
```